spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -876,6 +875,8 @@ def _clear_recording_region(
     def get_units(self, variable):
         if variable == NeuronRecorder.SPIKES:
             return NeuronRecorder.SPIKES
+        if variable == NeuronRecorder.PACKETS:
+            return ""
```

Comment:
count was the correct thing here

Reply:
I think "" == COUNT

Reply:
count is a different neo metric. whereas "" is unitless. so should be count me thinks